.github/workflows/acceptance_test.yml

```diff
@@ -48,62 +55,62 @@ jobs:
     needs: pre_ci
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: gradle/wrapper-validation-action@v1
+      - uses: actions/checkout@v4
+      - uses: gradle/actions/wrapper-validation@v3
   pack-snapshot:
     needs: [ validate-gradle-wrapper ]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up JDK 11
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
           java-version: '11'
           distribution: 'temurin'
       - name: Cache Gradle packages
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: ${{ runner.os }}-gradle
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v3
```

Comment:
Is there anything wrong with gradle/gradle-build-action@v2?

Reply:
Yes it's deprecated -- To fix all warnings it had to be updated. Any concerns with that?

Reply:
No concerns. Not much choice, is there?